build.bat

```diff
@@ -36,7 +38,9 @@ exit /B 0
 if "%1" == "--help" goto print_usage
 if "%1" == "--test" goto test
 if "%1" == "--testUi" goto testUi
+if "%1" == "--installCore" goto installCore
 if "%1" == "--packageJmc" goto packageJmc
+if "%1" == "--packageAgent" goto packageAgent
 if "%1" == "--clean" goto clean
```

Comment:
Lacks the --runAgentExample and the --runAgentConverterExample.

Reply:
because as I've mentioned above, maybe here can help someone with win machine. to parse properly the java version and base on that execute the agents. I bet the only parsing java version will be separate bat script. I'm not sure wether such option is worth to the effort.

Reply:
I've add one for java 11, i think it should be fine as you've said.